source/shared/cpp/ObjectModel/BaseActionElement.cpp

```diff
@@ -45,3 +48,15 @@ Json::Value BaseActionElement::SerializeToJsonValue()
     root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Id)] = GetId();
     return root;
 }
+
+void BaseActionElement::GetAdditionalProperty(std::string name, Json::Value & json)
```

Comment:
>void [](start = 0, length = 4)

Do you think it would make sense to make this a bool depending on whether the item was found?

Reply:
I like returning the json value since that's consistent with what the winrt layer will look like.  I'll change to that.

---
In reply to: [142794093](https://github.com/Microsoft/AdaptiveCards/pull/745#discussion_r142794093) [](ancestors = 142794093)